### PR TITLE
Deprecate/delete deploy box

### PR DESCRIPTION
**Pull request by stvstnfrd**
This PR replaces and removes the use of the old `deploy` boxes, off-loading its tasks to the build/jenkins box.
It does this in three steps, each in its own commit; please refer to commit messages for additional information.

I'm not a huge fan of the role's name, `stanford_deploy`, so it's still kind of a placeholder...

### CODE REVIEW

File: stanford/roles/stanford_deploy/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (dcadams) [inline stanford/roles/stanford_deploy/defaults/main.yml]: Yes, I'm not keen on the name either. How about 'extra_build_role'.

**Comment by dcadams** [comment]
BTW, the default of CLUSTER_INSTANCE_TYPE being t2.micro for lagunita is inadequate. The dump_dbs script took over 1 day just for Mongo. The machine it is replacing is a t2.2xlarge.

File: stanford/playbooks/build.yml [hunk 2]
```diff
@@ -2,16 +2,19 @@
...
   gather_facts: True
   become: true
   vars:
+    ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p {{ ansible_jumpbox }}"'
     CLUSTER_NAME: build
     jenkins_server_name: '_'
   vars_files:
     - "{{ COMMON_DEPLOYMENT_DIR }}/common.yaml"
   roles:
+    - deploy_manually
```
Comment (stvstnfrd) [inline stanford/playbooks/build.yml]: I moved `common` into an actual role dependency.

File: stanford/roles/deploy_manually/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+DEPLOY_MANUALLY_PATH_CONFIG: /opt/configuration
```
Comment (stvstnfrd) [inline stanford/roles/deploy_manually/defaults/main.yml]: `DEPLOY_MANUALLY` is the most literal interpretation of this role and why it exists,
which is to say, to run the deploy scripts manually.

File: stanford/roles/deploy_manually/meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - common
```
Comment (stvstnfrd) [inline stanford/roles/deploy_manually/meta/main.yml]: We need a few tasks from here, namely user creation.

**Comment by stvstnfrd** [comment]
@dcadams Ping.

**Comment by dcadams** [comment]
:+1:

**Pull request merged**